podcasts/FadeOutManager.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+
+class FadeOutManager {
```

Comment:
I don't like this name at all, but couldn't think of something better.

Reply:
Tricky one maybe:`VolumeControl` or `FaderMixer`